source/full_shutdown.rst

```diff
@@ -0,0 +1,206 @@
+.. include:: vars.rst
+
+=======================
+Full Shutdown Procedure
+=======================
+
+In case a full shutdown of the system is required, we advise to use the
+following order:
+
+* Perform a graceful shutdown of all virtual machine instances
+* Stop Ceph (if applicable)
```

Comment:
This might be early for stopping Ceph, in case the OpenStack services are still using Ceph state (eg, image uploads).  Perhaps stop Ceph at the point where the Ceph nodes are shut down.